recipes/r-st/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = '1.2.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-st
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: st_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/st_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/st/st_{{ version }}.tar.gz
+  sha256: eb020c1ee0a043f5e76d3dd781219eb409fe42584b657bd096f29df19974de36
+
+build:
+  number: 0
+  # r-fdrtool fails to build on windows
+  skip: true  # [win]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-corpcor >=1.6.8
+    - r-fdrtool >=1.2.15
+    - r-sda >=1.3.7
+  run:
+    - r-base
+    - r-corpcor >=1.6.8
+    - r-fdrtool >=1.2.15
+    - r-sda >=1.3.7
+
+test:
+  commands:
+    - $R -e "library('st')"  # [not win]
+    - "\"%R%\" -e \"library('st')\""  # [win]
+
+about:
+  home: http
```

Comment:
here is missing something....

Reply:
Oops, usually I catch those... conda_r_skeleton_helper leaves that empty sometimes.